server/service/service_appconfig.go

```diff
@@ -319,3 +320,7 @@ func (svc *Service) SetupRequired(ctx context.Context) (bool, error) {
 	}
 	return false, nil
 }
+
+func (svc *Service) FleetConfig(ctx context.Context) config.FleetConfig {
+	return svc.config
```

Comment:
Whatever we decide to call this method, let's add an explicit authorization check:
```suggestion
func (svc *Service) FleetConfig(ctx context.Context) config.FleetConfig {
	if err := svc.authz.Authorize(ctx, &fleet.AppConfig{}, fleet.ActionRead); err != nil {
		return nil, err
	}
	return svc.config
```
In the current usage, this is already covered by another authorization check in the API endpoint, but this would make it more resistant to errors during refactoring.